alf/algorithms/algorithm.py

```diff
@@ -1479,7 +1482,8 @@ def train_from_replay_buffer(self, update_global_counter=False):
         # training is not started yet, ``_replay_buffer`` will be None since it
         # is only lazily created later when online RL training started.
         if (self._replay_buffer and self._replay_buffer.total_size
-                < config.initial_collect_steps):
+                < config.initial_collect_steps) or (effective_unroll_steps
+                                                    == 0):
```

Comment:
Is there any situation that ``train_from_replay_buffer`` will be called with ``effective_unroll_steps=0``?

Reply:
effective_unroll_steps is now removed from this function